From 48ee1008f59eed95cbca7a8b1b3630ddbffdaae5 Mon Sep 17 00:00:00 2001
From: Terry Wilson <twilson@redhat.com>
Date: Fri, 24 Jul 2020 16:42:17 -0500
Subject: [PATCH] Fix thread/threading change in unit test

This unit test was not updated when the thread/threading name
change happened. This fixes the unit test when eventlet is
installed.

Change-Id: I0e0da83187190c7145bb01f77daf22a43868115c
Closes-bug: #1875678
---
 ovsdbapp/tests/unit/test_api.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/ovsdbapp/tests/unit/test_api.py b/ovsdbapp/tests/unit/test_api.py
index 897a955..8bb7921 100644
--- a/ovsdbapp/tests/unit/test_api.py
+++ b/ovsdbapp/tests/unit/test_api.py
@@ -44,12 +44,12 @@ except ImportError:
 class GreenThreadingFixture(fixtures.Fixture):
     def _setUp(self):
         if 'eventlet' in sys.modules:
-            self._orig = api.thread.get_ident
-            api.thread.get_ident = thread.get_ident
+            self._orig = api.threading.get_ident
+            api.threading.get_ident = thread.get_ident
             self.addCleanup(self.cleanup)
 
     def cleanup(self):
-        api.thread.get_ident = self._orig
+        api.threading.get_ident = self._orig
 
 
 class FakeTransaction(object):
-- 
2.27.0

